src/main/resources/referenceImageTemplateWithScript.json

```diff
@@ -105,7 +105,7 @@
                     "osDisk": {
                         "name": "osdisk",
                         "vhd": {
-                            "uri": "[concat('http://',variables('storageAccountName'),'.blob.core.windows.net/',variables('storageAccountContainerName'),'/', variables('vmName'), copyIndex(), 'OSDisk.vhd')]"
+                            "uri": "[concat('http://',variables('storageAccountName'),'.blob.core.windows.net/',variables('storageAccountContainerName'),'/', variables('vmName'), uniquestring(resourceGroup().id), 'OSDisk.vhd')]"
```

Comment:
sorry guys .. this change was unintentional .. don't know when and how .. I will have to remove it ... thanks for catching